### DISCUSSION TITLE
I really can't make TALMemo work or WebRTC :(

### DISCUSSION THREAD

**Discussion by magnolima** [MagicFoundation/Alcinoe#170] [Q&A]
I have to fix the DX.bat to ensure the **librariesRootPath** correctly find the ..alcinoe-master\lib\jar dir. Also the line:
**SET _d8Cmd=%_dxCmd:dx.bat"=d8.bat"!_extraParams! --lib** _C:\Android-SDK\platforms\android-29\android.jar%_ (my sdk path)

Before that I was unable to compile. Reading previous issues posts I figured out that I should include the patched FMX that should be into the ..\alcinoe-master\references\embarcadero  folder, but it is no longer valid, the Embarcadero folder is moved to \alcinoe-master\embarcadero\{my delphi version} folder and btw I am using the 10.3.3 CE edition.

Also I need to deploy some .jar libs (I dunno exactly what these jar files are), but I have a doubt if the deploy meant to add into the Project -> Deployment -> Add files (what could be the target folder there? /res? ) or adding via Target->Libraries (which I think that could be the correct).

The project compiles but it will crash as soon it opens.

**Comment by Zeus64** [comment]
do you have any error message when it's crash ?

**Reply by magnolima** [reply]
No message, it crashes even before the Delphi's debug have the chance catch the error.

I added the libs alcinoe-edittext.jar and alcinoe-common.jar to  Target->Libraries and also disabled the android-support-v4.dex.jar.

**Reply by Zeus64** [reply]
Ok tomorow I will make a small test on Rio 10.3.3 but I think as the demo dproj file was updated for sydney this make a mess when launching it from rio.

**Reply by magnolima** [reply]
Thank you very much for it! Also, I was not able to find a TALMemo demo.
